language.cc

```diff
@@ -255,7 +255,7 @@ Db::Db()
   iso2ToCountry[ "bo" ] = "cn";
   iso2ToCountry[ "br" ] = "fr";
   iso2ToCountry[ "bs" ] = "ba";
-  iso2ToCountry[ "ca" ] = "ad";
+  iso2ToCountry[ "ca" ] = "catalonia";
```

Comment:
According to Wikipedia:
* "ad" is the language code of Andorra;
* Andorra is a sovereign country;
* Catalan is the official language in Andorra;
* Catalonia is an autonomous community of Spain.

On the other hand, the population of Catalonia is almost 100 times the population of Andorra.

Therefore, I am not sure about this change.